CHANGES.md

```diff
@@ -1,5 +1,21 @@
 # Changes
 
+## edge-21.4.5
+
+This edge release adds a new `--short` flag to `linkerd check` to show a
+summary of the check output. This release also includes various proxy bug fixes
+and improvements.
+
+* Fixes a task leak that could be triggered when clients disconnect when
```

Comment:
What do you think about changing the verbs here to a past tense? Think it'd be a bit more consistent with the rest of the log.